sql/analyzer/validate_create_table.go

```diff
@@ -54,10 +54,6 @@ func validateCreateTable(ctx *sql.Context, a *Analyzer, n sql.Node, scope *Scope
 }
 
 func validateAlterColumn(ctx *sql.Context, a *Analyzer, n sql.Node, scope *Scope, sel RuleSelector) (sql.Node, transform.TreeIdentity, error) {
```

Comment:
maybe we should call this `resolveAlterColumn` now

Reply:
Good call